### DISCUSSION TITLE
Seems like you don't have to manually install react-is

### DISCUSSION THREAD

**Discussion by Axedyson** [urql-graphql/urql#2058] [Q&A] [answered]
In the docs about how to setup urql with next.js the following is stated:
```
yarn add next-urql react-is urql graphql
# or
npm install --save next-urql react-is urql graphql
```
However I tried to not install `react-is` and everything seems to work fine. Also when I look in my node_modules folder I see `react-is` installed, without me doing it manually. I'm using yarn 3.0.2. So maybe the docs should be updated to exclude `react-is` as part of the necessary packages to install

**Comment by nandorojo** [comment, marked as answer]
Run `yarn why react-is`. Is it possible a different package had it installed?

**Reply by Axedyson** [reply]
This is what I get:
![image](https://user-images.githubusercontent.com/28491857/140382486-81494f9f-b402-4e05-9396-004e913a8c83.png)

**Reply by nandorojo** [reply]
Yeah, so it looks like you already had it from other libraries, which is why you didn’t need to install it

**Reply by Axedyson** [reply]
Okay would you still say it's good practice to have the package written down in `package.json` to be safe that it always works?

**Reply by nandorojo** [reply]
Yeah probably. That way, any libs that have it as a direct dependency will turn into their nested dependency. And any libs that have it as a peer/dev dependency (like urql) I think will pull it from the version you define in your own package.json.

**Reply by nandorojo** [reply]
You can confirm my assumption by installing it yourself, then open `node_modules/next/node_modules` and ensure the version listen from `yarn why` is there.
